charts/node/values.yaml

```diff
@@ -338,6 +338,10 @@ node:
         enabled: false
         hostname: example.com
         ttl: 300
+      # If enabled, additionally expose WebSocket port. Useful for bootnodes
+      ws:
+        enabled: false
+        port: 30334
```

Comment:
This port needs to be different from `paraP2pService.port` (TCP p2p) and `relayP2pService.port`. I would suggest 30444.

Reply:
I updated the ports:
- relaychain-p2p: 30333 (remained unchanged)
- relaychain-p2p-ws: 30334 (remained unchanged)
- parachain-p2p: 30444
- parachain-p2p-ws: 30445

Reply:
Never mind, it seems that they are fixed before I posted

Reply:
Related: https://github.com/paritytech/helm-charts/issues/178